ci-operator/step-registry/cucushift/upgrade/arch-migration/cucushift-upgrade-arch-migration-commands.sh

```diff
@@ -292,6 +312,19 @@ function health_check() {
     check_pod
 }
 
+# https://issues.redhat.com/browse/OTA-961
+function check_co() {
```

Comment:
the function name is a bit confused to users, maybe `check_mco_image` ?

Reply:
I am okay with either of options.

Reply:
Removed this function.